src/main/java/com/spotify/reaper/cassandra/JmxProxy.java

```diff
@@ -272,7 +277,7 @@ public int getPendingCompactions() {
       } catch (InstanceNotFoundException e) {
         // This happens if no repair has yet been run on the node
         // The AntiEntropySessions object is created on the first repair
-        LOG.debug("No compaction has run yet on the node. Ignoring exception.", e);
+        LOG.error("Error getting pending compactions attribute from JMX", e);
         return 0;
       } catch (Exception e) {
         LOG.error("Error getting attribute from JMX", e);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Error getting attribute from JMX" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)